cli/dts/lib.deno.ns.d.ts

```diff
@@ -86,93 +112,225 @@ declare interface PerformanceMeasureOptions {
+   *   }
+   * }
+   * ```
    *
    * @category Errors
    */
   export namespace errors {
-    /** @category Errors */
+    /**
+     * Raised when the underlying operating system indicates that the file
+     * was not found.
+     *
+     * @category Errors */
     export class NotFound extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when the underlying operating system indicates the current user
+     * which the Deno process is running under does not have the appropriate
+     * permissions to a file or resource.
+     *
+     * @category Errors */
     export class PermissionDenied extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when the underlying operating system reports that a connection to
+     * a resource is refused.
+     *
+     * @category Errors */
     export class ConnectionRefused extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when the underlying operating system reports that a connection has
+     * been reset. With network servers, it can be a _normal_ occurrence where a
+     * client will abort a connection instead of properly tearing down the
+     * connection.
+     *
+     * @category Errors */
     export class ConnectionReset extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when the underlying operating system reports an `ECONNABORTED`
+     * error.
+     *
+     * @category Errors */
     export class ConnectionAborted extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when the underlying operating system reports an `ENOTCONN` error.
+     *
+     * @category Errors */
     export class NotConnected extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when attempting to open a server listener on an address and port
+     * that already has a listener.
+     *
+     * @category Errors */
     export class AddrInUse extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when the underlying operating system reports an `EADDRNOTAVAIL`
+     * error.
+     *
+     * @category Errors */
     export class AddrNotAvailable extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when trying to write to a resource and a broken pipe error occurs.
+     * This can happen when trying to write directly to `stdout` or `stderr`
+     * and the operating system is unable to pipe the output for a reason
+     * external to the Deno runtime.
+     *
+     * @category Errors */
     export class BrokenPipe extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when trying to create a resource, like a file, that already
+     * exits.
+     *
+     * @category Errors */
     export class AlreadyExists extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when trying to internally convert data from an resource into a
+     * string. Deno expects source data to be encoded as UTF-8 and will error
+     * if the data is not valid UTF-8.
```

Comment:
This is raised by the OS, so no guaruantee it is related to UTF-8 things. We should make it more generic: "Raised when an operation to returns data that is invalid for the operation being performed."